### DISCUSSION TITLE
请问网站里不规则盒子怎么都做到排版整齐划一的

### DISCUSSION THREAD

**Discussion by HunterX-327** [jaywcjlove/reference#846] [Q&A]
因为是reference每条功能都有不确定性的内容大小，但是网站排版基本都是对齐的感觉很厉害，不论是怎么样的几个list排版在一起。
具体是使用了grid布局吗，有没有其他技巧呢？

**Comment by jaywcjlove** [comment]
grid 布局，然后调整内容
